libvcx/src/aries/handlers/connection/connection.rs

```diff
@@ -87,7 +95,7 @@ impl Connection {
         let pairwise_info = PairwiseInfo::create()?;
         let mut connection = Connection {
             cloud_agent_info: CloudAgentInfo::default(),
-            connection_sm: SmConnection::Invitee(SmConnectionInvitee::new(source_id, pairwise_info)),
+            connection_sm: SmConnection::Invitee(SmConnectionInvitee::new(source_id, pairwise_info, send_message)),
```

Comment:
... but now this is not consistent (following up on my previous comment)

Reply:
the above was inlined